### PR TITLE
Bdd

### PR DESCRIPTION
**Pull request by phlptt**

### CODE REVIEW

File: ui_automation_tests/conftest.py [hunk 1]
```diff
@@ -42,200 +42,112 @@ def driver(request):
...
+
+
+@pytest.fixture(scope="module")
+def sso_sign_in_url(request):
+     return request.config.getoption("--sso_sign_in_url")
+
 
 @pytest.fixture(scope="module")
 def invalid_username():
     return "invalid@mail.com"
```
Comment (lukesa) [inline ui_automation_tests/conftest.py]: should it be a CONST?

File: ui_automation_tests/conftest.py [hunk 2]
```diff
@@ -42,200 +42,112 @@ def driver(request):
...
-def internal_login_url():
-    return "https://sso.trade.uat.uktrade.io/login/"
 
 @pytest.fixture(scope="function")
-def open_internal_hub(driver, internal_url, sign_in_url):
-    driver.get(sign_in_url)
+def open_internal_hub(driver, internal_url, sso_sign_in_url):
+    driver.get(sso_sign_in_url)
     driver.find_element_by_name("username").send_keys("test-uat-user@digital.trade.gov.uk")
     driver.find_element_by_name("password").send_keys("5cCIlffSrqszgOuw23VEOECnM")
```
Comment (lukesa) [inline ui_automation_tests/conftest.py]: both user/pass should come from .env

File: ui_automation_tests/conftest.py [hunk 3]
```diff
@@ -42,200 +42,112 @@ def driver(request):
...
+    driver.get(internal_url)
 
 
 @given('I go to internal homepage')
-def go_to_internal_homepage(driver, internal_url, internal_login_url):
-    driver.get(internal_login_url)
+def go_to_internal_homepage(driver, internal_url, sso_sign_in_url):
+    driver.get(sso_sign_in_url)
     driver.find_element_by_name("username").send_keys("test-uat-user@digital.trade.gov.uk")
     driver.find_element_by_name("password").send_keys("5cCIlffSrqszgOuw23VEOECnM")
```
Comment (lukesa) [inline ui_automation_tests/conftest.py]: both user/pass should come from .env

**Comment by phlptt** [comment]
@lukesa please rereview

File: ui_automation_tests/conftest.py [hunk 4]
```diff
@@ -42,200 +43,116 @@ def driver(request):
...
 
 @pytest.fixture(scope="module")
 def invalid_username():
     return "invalid@mail.com"
 
-@pytest.fixture(scope="module")
-def internal_login_url():
-    return "https://sso.trade.uat.uktrade.io/login/"
+
+sso_email="test-uat-user@digital.trade.gov.uk"
```
Comment (cstobart) [inline ui_automation_tests/conftest.py]: probably best to store these in vault

**Pull request merged**